### ISSUE TITLE
stuck at the first step.

### ISSUE THREAD

**Issue by umb-c** [kijai/ComfyUI-FluxTrainer#202]
I don't know what happened of what changed, it was working fine till yesterday. Today everything i try it get stuck at the first step. it stays like that and do nothing. 

 Loading state dict from          flux_utils.py:114
                             C:\Users\stizz\Documents\ComfyUI                  
                             \models\diffusion_models\flux1-d                  
                             ev-fp8-e4m3fn.safetensors                         
                    INFO     Loaded Flux: <All keys matched   flux_utils.py:132
                             successfully>                                     
                    INFO     Loaded              flux_train_network_comfy.py:69
                             torch.float8_e4m3fn                               
                             FLUX model                                        
                    INFO     Building CLIP-L                  flux_utils.py:158
                    INFO     Loading state dict from          flux_utils.py:254
                             C:\Users\stizz\Documents\ComfyUI                  
                             \models\text_encoders\clip_l.saf                  
                             etensors                                          
                    INFO     Loaded CLIP-L: <All keys matched flux_utils.py:257
                             successfully>                                     
                    INFO     Loading state dict from          flux_utils.py:309
                             C:\Users\stizz\Documents\ComfyUI                  
                             \models\text_encoders\google_t5-                  
                             v1_1-xxl_encoderonly-fp8_e4m3fn.                  
                             safetensors                                       
                    INFO     Loaded T5xxl: <All keys matched  flux_utils.py:312
                             successfully>                                     
                    INFO     Loaded fp8 T5XXL    flux_train_network_comfy.py:94
                             model                                             
                    INFO     Building AutoEncoder             flux_utils.py:139
                    INFO     Loading state dict from          flux_utils.py:144
                             C:\Users\stizz\Documents\ComfyUI                  
                             \models\vae\ae.safetensors                        
                    INFO     Loaded AE: <All keys matched     flux_utils.py:147
                             successfully>                                     
import network module: .networks.lora_flux
                    INFO     [Dataset 0]                     train_util.py:2479
                    INFO     caching latents with caching    train_util.py:1026
                             strategy.                                         
                    INFO     caching latents...              train_util.py:1075
100%|██████████| 33/33 [00:13<00:00,  2.49it/s]
2025-11-24 20:08:29 INFO     move vae and unet  flux_train_network_comfy.py:164
                             to cpu to save                                    
                             memory                                            
                    INFO     move text encoders flux_train_network_comfy.py:172
                             to gpu                                            
2025-11-24 20:08:30 INFO     prepare T5XXL for  flux_train_network_comfy.py:472
                             fp8: set to                                       
                             torch.float8_e4m3f                                
                             n, set embeddings                                 
                             to torch.bfloat16,                                
                             add hooks                                         
                    INFO     [Dataset 0]                     train_util.py:2501
                    INFO     caching Text Encoder outputs    train_util.py:1209
                             with caching strategy.                            
                    INFO     checking cache validity...      train_util.py:1220
100%|██████████| 33/33 [00:00<00:00, 32994.52it/s]
                    INFO     caching Text Encoder outputs... train_util.py:1251
  0%|          | 0/33 [00:00<?, ?it/s]                    WARNING  T5 model is using fp8 weights strategy_flux.py:151
                             for caching. This may affect                      
                             the quality of the cached                         
                             outputs.                                          
100%|██████████| 33/33 [00:05<00:00,  5.58it/s]
2025-11-24 20:08:36 INFO     cache Text Encoder flux_train_network_comfy.py:188
                             outputs for sample                                
                             prompt: ['naomi is                                
                             a woman']                                         
                    INFO     cache Text Encoder flux_train_network_comfy.py:218
                             outputs for                                       
                             prompt: naomi is a                                
                             woman                                             
                    INFO     cache Text Encoder flux_train_network_comfy.py:218
                             outputs for                                       
                             prompt:                                           
2025-11-24 20:08:37 INFO     move CLIP-L back   flux_train_network_comfy.py:229
                             to cpu                                            
                    INFO     move t5XXL back to flux_train_network_comfy.py:231
                             cpu                                               
2025-11-24 20:08:38 INFO     move vae and unet  flux_train_network_comfy.py:236
                             back to original                                  
                             device                                            
                    INFO     create LoRA network. base dim     lora_flux.py:491
                             (rank): 16, alpha: 1.0                            
                    INFO     neuron dropout: p=None, rank      lora_flux.py:492
                             dropout: p=None, module dropout:                  
                             p=None                                            
                    INFO     train all blocks only             lora_flux.py:502
                    INFO     create LoRA for Text Encoder 1:   lora_flux.py:589
                    INFO     create LoRA for Text Encoder 1:   lora_flux.py:592
                             72 modules.                                       
                    INFO     create LoRA for FLUX all blocks:  lora_flux.py:606
                             304 modules.                                      
                    INFO     enable LoRA for U-Net: 304        lora_flux.py:755
                             modules                                           
FLUX: Gradient checkpointing enabled. CPU offload: False
prepare optimizer, data loader etc.
                    INFO     use CAME optimizer | {}         train_util.py:4882
enable full bf16 training.
enable fp8 training for U-Net.
unet_weight_dtype: torch.float8_e4m3fn
                    INFO     set U-Net weight dtype to     train_network.py:638
                             torch.float8_e4m3fn                               
running training
  num train images * repeats: 330
  num reg images: 0
  num batches per epoch: 330
  num epochs: 10
  batch size per device: 1
  gradient accumulation steps: 1
  total optimization steps: 3000
2025-11-24 20:08:49 INFO     text_encoder is not needed   train_network.py:1107
                             for training. deleting to                         
                             save memory.                                      
                    INFO     unet dtype:                  train_network.py:1124
                             torch.float8_e4m3fn, device:                      
                             cuda:0                                            
Epoch 1/10 - steps:   0%|          | 0/3000 [00:00<?, ?it/s]
